spa/cypress/integration/01 - donation-page.spec.js

```diff
@@ -19,7 +20,7 @@ describe('Routing', () => {
   it('should send a request containing the correct query params', () => {
     cy.intercept({ method: 'GET', pathname: getEndpoint(LIVE_PAGE_DETAIL) }, (req) => {
       expect(req.url).contains(`revenue_program=${EXPECTED_RP_SLUG}`);
-      expect(req.url).contains(`page=${expectedPageSlug}`);
+      expect(req.url).contains(`page=${pageSlug}`);
```

Comment:
Shouldn't this one continue to be `expectedPageSlug`?
```suggestion
      expect(req.url).contains(`page=${expectedPageSlug}`);
```

Reply:
`expectedPageSlug` is a route and `pageSlug` is the name of page which we are getting from backend.
This should be `pageSlug`only.